### DISCUSSION TITLE
How to block Disqus on a specific page?

### DISCUSSION THREAD

**Discussion by Acorn2Oak** [datenstrom/community#657] [Ask a question] [answered]
I am looking for guidance on the best approach to blocking Discus on a specific Blog page. I had expected  sometime like [disqus off] or [disqus 0].

Appreciated

**Comment by schulle4u** [comment, marked as answer]
Thanks for your inspiration. The [Disqus extension](https://github.com/datenstrom/yellow-extensions/tree/master/source/disqus) has been updated to support pages without comments. Here is an example content file to prevent loading the Disqus thread. 

```
---
Title: Example page
Comment: exclude
---
This page does not show comments.
```

I hope this solves your problem.

**Reply by Acorn2Oak** [reply]
Simple when you think it through but nonetheless a very useful snippet for others.

**Reply by Acorn2Oak** [reply]
> Simple when you think it through but nonetheless a very useful snippet for others.

Just added to a Blog page and, sadly, nothing changed.
However, I was too quick as the Disqus update did not engage immediately as my Internet coughed!

**All fine**
